packages/twenty-front/src/modules/prefetch/components/PrefetchRunQueriesEffect.tsx

```diff
@@ -42,7 +45,7 @@ export const PrefetchRunQueriesEffect = () => {
 
   const { result } = useCombinedFindManyRecords({
     operationSignatures,
-    skip: !currentUser,
+    skip: !currentUser || isSettingsPage,
```

Comment:
sorry, some miscomunication here: we should skip if activationStatus == SUSPENDED here

Reply:
but I would not check useIsPrefetchLoading in the NavItem in case of settingsPage instead